### PR TITLE
Releases: add 0.18.1

### PR DESCRIPTION
**Pull request by harding**
- [x] Update `_releases/0.18.1.md` with the actual final release date, the actual final release notes, and (optionally) add the magnet link (uncommenting that field).
- [x] Update _posts/en/posts/2019-08-02-release-0.18.1.md by renaming its base name to start with the actual final release date

Reminder: Travis is expected to fail on release PRs until final 0.18.1 binaries have been uploaded to the bitcoincore.org/bin to ensure that our download links work.  Just restart Travis when everything is ready.

Note: the upstream release notes contained a link that pointed to a missing systemd section.  I think these are an unnecessary holdover from the 0.18.0 release notes and opened https://github.com/bitcoin/bitcoin/pull/16532 to address the issue upstream.  However, to fix the broken link here, I removed just the sentence with the link; I haven't edited the release notes in any other way.

### CODE REVIEW

File: _releases/0.18.1.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+# Release notes from bitcoin/bitcoin 784e218610d4dab07e3523b71566cb2516c04d2c ("doc: Update release notes for rc1", 2019-07-24)
+---
+{% include download.html %}
+{% githubify https://github.com/bitcoin/bitcoin %}
+Bitcoin Core version 0.18.1 is now available from:
+
+  <https://bitcoincore.org/bin/bitcoin-core-0.18.1/>
+
+This is a new major version release, including new features, various bug
```
Comment (azuchi) [inline _releases/0.18.1.md]: `minor` version is correct.

Comment (harding) [reply _releases/0.18.1.md]: Fixed here and upstream.  Thanks!

**Comment by laanwj** [comment]
Thank you!

**Comment by laanwj** [comment]
ACK 1a7deffcf7a45bce132d72b38389ad041b7fe4c2 + b6b2dc13d7e118009615685e9570cc81344a5cbc

**Pull request merged**